app/helpers.php

```diff
@@ -1,7 +1,47 @@
 <?php
 
 /**
- * Parse a CSV field value as boolean.
+ * Returns import type for Facebook share files.
+ * @return string
+ */
+function get_import_type_facebook()
```

Comment:
Let's make a `ImportType` class that exports these constants:

```php
// app/Import.php
namespace Chompy;

class ImportType {
  public static $facebook = 'facebook';
  public static $rockTheVote = 'rock-the-vote';
  public static $turbovote = 'turbovote';

  public static function all() {
    return [self::$facebook, self::$rockTheVote, self::$turbovote];
  }
}
```

We can then use these anywhere else with `\Chompy\ImportType::$turbovote` or `\Chompy\ImportType::all()` without cluttering our helpers file with tons of constants over time.